### ISSUE TITLE
Can not update a single value in an embedded list  

### ISSUE THREAD

**Issue by jiujiu1123** [orientechnologies/orientdb#3328]
Hi I have a record which have a embedded list of string, I try to update a single value with this sql command.

``` sql
update #18:0 SET images[0] ='www.someurl.com/pic.jpg'
```

no error was thrown but the property was not updated.
I used this command to check.

``` sql
select expand(images) from #18:0
```

is there another way of updating a single value in an embedded list, or updating the entire list is required?

**Comment by lvca** [comment]
Unfortunately this isn't supported by SQL API, but we absolutely need it.

**Comment by Eric24** [comment]
+1

**Comment by luigidellaquila** [comment]
It's already fixed in v 3.0

Closing, thanks

Luigi
